### DISCUSSION TITLE
Playdead's INSIDE

### DISCUSSION THREAD

**Discussion by DragonTake** [skyline-emu/skyline#1123] [Game Requests]
Please add this game to the list

**Comment by PixelyIon** [comment]
I'll be curbing this request to just INSIDE as Limbo is made on Playdead's custom Box2D based engine while INSIDE is made in Unity. You'll have to create a separate request for Limbo.

**Reply by Sofiene34080** [reply]
Please miitopia 🤗🤗🤗🤗
